Gemfile

```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in ombu_labs-auth.gemspec.
 gemspec
 
-gem 'sqlite3', '~> 1.5', '>= 1.5.3'
+gem "sqlite3", "~> 2.6"
```

Comment:
Added native support for Ruby 3.4 [v2.4.1](https://github.com/sparklemotion/sqlite3-ruby/blob/main/CHANGELOG.md#250--2024-12-25)